src/webportal/src/app/user/fabric/user-profile/storage-list.jsx

```diff
@@ -37,17 +37,18 @@ function normalizePath(path) {
 
 function getStorageServerUri(server) {
   let blobAddress;
+  let fileAddress;
   const data = server.data;
   switch (server.type) {
     case 'nfs':
       return `nfs://${data.address}/${normalizePath(data.rootPath)}`;
     case 'samba':
       return `smb://${data.address}/${normalizePath(data.rootPath)}`;
     case 'azurefile':
-      return `smb://${data.dataStore}/${normalizePath(data.fileShare)}`;
+      fileAddress = `${data.accountName}.file.core.windows.net`;
+      return `smb://${fileAddress}/${normalizePath(data.fileShare)}`;
     case 'azureblob':
-      blobAddress =
-        data.dataStore || `${data.accountName}.blob.core.windows.net`;
+      blobAddress = `${data.accountName}.blob.core.windows.net`;
       return `http://${blobAddress}/${normalizePath(data.containerName)}`;
```

Comment:
where will this address be used? for blobfuse and strorage explorer, two parameters storage account and container are needed

Reply:
Just want to show `accountName` and `containerName` here. Any recommend representation? `accountName:containerName/path` ?

Reply:
Keep this format and change the `blob.core.windows.net` to `<AZURE_STORAGE_DNS_SUFFIX>` since different region has different suffix. And we just want to show `account name` and `container name` here